SystemServices/SystemServices.cpp

```diff
@@ -2216,6 +2218,126 @@ namespace WPEFramework {
 		returnResponse(resp);
 	}
 
+
+	uint32_t SystemServices::setTerritory(const JsonObject& parameters, JsonObject& response)
+	{
+		bool resp = false;
+		if(parameters.HasLabel("territory")){
+			if(!Utils::fileExists(TERRITORYFILE)){
+				LOGWARN(" Territory : Subdirectories created : %d",val );
+			}
+			readTerritoryFromFile();
```

Comment:
Do we need to read the file? We would just have to write into it.

Reply:
we should get the oldTerritory from file